R/utils.R

```diff
@@ -80,13 +78,18 @@ dots <- function(...) {
   eval(substitute(alist(...)))
 }
 
-slug <- function(x, ext) {
+asciify <- function(x) {
   stopifnot(is.character(x))
 
   x <- tolower(x)
-  x <- gsub("[^a-z0-9_]+", "-", x)
+  gsub("[^a-z0-9_]+", "-", x)
```

Comment:
I wonder if we should include `-` in the character class, right now we are substituting `-` with itself, e.g. change the regex to

```r
gsub("[^a-z0-9_-]+", "-", x)
```